src/main/java/net/minecraftforge/gradle/user/UserBasePlugin.java

```diff
@@ -1,22 +1,58 @@
-import net.minecraftforge.gradle.tasks.abstractutil.ExtractTask;
-import net.minecraftforge.gradle.tasks.user.SourceCopyTask;
-import net.minecraftforge.gradle.tasks.user.reobf.ArtifactSpec;
-import net.minecraftforge.gradle.tasks.user.reobf.ReobfTask;
-import org.gradle.api.*;
+import static net.minecraftforge.gradle.common.Constants.FERNFLOWER;
+import static net.minecraftforge.gradle.common.Constants.JAR_CLIENT_FRESH;
+import static net.minecraftforge.gradle.common.Constants.JAR_MERGED;
+import static net.minecraftforge.gradle.common.Constants.JAR_SERVER_FRESH;
+import static net.minecraftforge.gradle.user.UserConstants.ASTYLE_CFG;
+import static net.minecraftforge.gradle.user.UserConstants.CLASSIFIER_DECOMPILED;
+import static net.minecraftforge.gradle.user.UserConstants.CLASSIFIER_DEOBF_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.CLASSIFIER_SOURCES;
+import static net.minecraftforge.gradle.user.UserConstants.CONFIG_DEPS;
+import static net.minecraftforge.gradle.user.UserConstants.CONFIG_MC;
+import static net.minecraftforge.gradle.user.UserConstants.CONFIG_NATIVES;
+import static net.minecraftforge.gradle.user.UserConstants.CONFIG_START;
+import static net.minecraftforge.gradle.user.UserConstants.CONFIG_USERDEV;
+import static net.minecraftforge.gradle.user.UserConstants.CONF_DIR;
+import static net.minecraftforge.gradle.user.UserConstants.DEOBF_MCP_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.DEOBF_SRG_MCP_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.DEOBF_SRG_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.DIRTY_DIR;
+import static net.minecraftforge.gradle.user.UserConstants.EXC_JSON;
+import static net.minecraftforge.gradle.user.UserConstants.EXC_MCP;
+import static net.minecraftforge.gradle.user.UserConstants.EXC_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.FIELD_CSV;
+import static net.minecraftforge.gradle.user.UserConstants.GRADLE_START_CLIENT;
+import static net.minecraftforge.gradle.user.UserConstants.GRADLE_START_SERVER;
+import static net.minecraftforge.gradle.user.UserConstants.MAPPING_APPENDAGE;
+import static net.minecraftforge.gradle.user.UserConstants.MCP_PATCH_DIR;
+import static net.minecraftforge.gradle.user.UserConstants.MERGE_CFG;
+import static net.minecraftforge.gradle.user.UserConstants.METHOD_CSV;
+import static net.minecraftforge.gradle.user.UserConstants.PACKAGED_EXC;
+import static net.minecraftforge.gradle.user.UserConstants.PACKAGED_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.PARAM_CSV;
+import static net.minecraftforge.gradle.user.UserConstants.RECOMP_CLS_DIR;
+import static net.minecraftforge.gradle.user.UserConstants.RECOMP_SRC_DIR;
+import static net.minecraftforge.gradle.user.UserConstants.REOBF_NOTCH_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.REOBF_SRG;
+import static net.minecraftforge.gradle.user.UserConstants.SOURCES_DIR;
```

Comment:
I might remove later all those import static and might set some code styles later, I just dont know if it's possible to make it compatible with eclipse

Reply:
I would just put a single `import static net.minecraftforge.gradle.user.UserConstants.*;` in place or all these different constant imports, or something of the like.